homeassistant/components/recorder/migration.py

```diff
@@ -168,7 +168,7 @@ class _ColumnTypesForDialect:
     big_int_type="INTEGER",
     timestamp_type="FLOAT",
     context_bin_type="BLOB",
-    small_int_type="INTEGER",
+    small_int_type="SMALLINT",
```

Comment:
Why is this changed? SQLITE has no distinct SMALLINT datatype: https://www.sqlite.org/datatype3.html